torch/_tensor_docs.py

```diff
@@ -4102,6 +4102,75 @@ def callable(a, b) -> number
     >>> torch.equal(b, c)
     False
 
+
+view(dtype) -> Tensor
+
+Returns a new tensor with the same data as the :attr:`self` tensor but of a
+different :attr:`dtype`.
+
+The returned tensor shares the same data and must have the same number
+of elements, but may have a different dtype. For a tensor to be viewed, the new
+dtype must have the same number of bytes with its original dtype.
+
+.. warning::
+
+    This overload is not supported by TorchScript yet. Confusing error message
+    like `RuntimeError: shape '[3]' is invalid for input of size 125` or silent
+    wrong result might be produced if you try to use this operator in TorchScript
+
+
+Args:
+    dtype (:class:`torch.dtype`): the desired dtype
+
+Example::
+
+    >>> x = torch.randn(4, 4)
+    >>> x
+    tensor([[-0.3113, -1.6569,  0.5822,  0.0424],
+            [ 1.1197, -1.5123, -0.8537, -1.9865],
+            [ 1.0786,  1.4348,  0.1338,  0.0893],
+            [-0.3087,  1.3192,  0.3953, -0.6418]])
+    >>> x.dtype
```

Comment:
This is cool but I wouldn't show the size or stride. Instead I suggest changing to int, updating the first int32 value, and then demonstrating the updated fp32 value. So the same presentation, minus the size/stride bit, plus an assignment after printing z. 

Nit: typically we use x and y or a and b when dealing with two tensors.

Reply:
fixed